server/routes_test.go

```diff
@@ -104,6 +105,24 @@ func Test_Routes(t *testing.T) {
 				assert.Empty(t, len(modelList.Models))
 			},
 		},
+		{
+			Name:   "openai list models no tags",
```

Comment:
Perhaps an empty list test?

Reply:
This test is an empty list test, can rename